### PR TITLE
Add DISABLE_BASIC_PARSER_STYLES define

### PR DESCRIPTION
**Pull request by fdarosa2663**

### CODE REVIEW

File: sound/hybrid_font.h [hunk 1]
```diff
@@ -21,6 +21,8 @@ class FontConfigFile : public ConfigFile {
     CONFIG_VARIABLE2(ProffieOSBlastImageDuration, 1000.0f);
     CONFIG_VARIABLE2(ProffieOSClashImageDuration, 500.0f);
     CONFIG_VARIABLE2(ProffieOSForceImageDuration, 1000.0f);
+    CONFIG_VARIABLE2(ProffieOSMinSwingAcceleration, 0.0f);
```
Comment (profezzorn) [inline sound/hybrid_font.h:24]: This doesn't belong in this PR

Comment (fdarosa2663) [reply sound/hybrid_font.h:24]: strange, maybe I branched wrong version, will resubmit.
